.github/workflows/_swift.yml

```diff
@@ -5,172 +5,79 @@ on:
 
 jobs:
   build:
-    name: build-${{ matrix.sdk }}
-    runs-on: ${{ matrix.runs-on }}
+    name: ${{ matrix.job_name }}
+    runs-on: macos-15
```

Comment:
This was updated to match our development machines which reduces the build configuration drift between the two.